Libraries/CustomComponents/NavigationExperimental/NavigationCardStack.js

```diff
@@ -221,16 +220,7 @@ class NavigationCardStack extends React.Component<DefaultProps, Props, void> {
   _configureTransition = () => {
     const isVertical = this.props.direction === 'vertical';
     const animationConfig = {};
-    if (
-      !!NativeAnimatedModule
-
-      // Gestures do not work with the current iteration of native animation
-      // driving. When gestures are disabled, we can drive natively.
-      && !this.props.enableGestures
-
-      // Native animation support also depends on the transforms used:
-      && NavigationCardStackStyleInterpolator.canUseNativeDriver(isVertical)
-    ) {
+    if (NavigationCardStackStyleInterpolator.canUseNativeDriver(isVertical)) {
```

Comment:
1. Why removed `!!NativeAnimatedModule`? then there will be a warning to add `NativeAnimation` library if they upgrade from a early version of RN if I don't miss the commits.
2. `NavigationCardStackStyleInterpolator.canUseNativeDriver` always returns `true` now as all the transforms can be driven by native, so I think we can make this function simpler .